lib/oban.ex

```diff
@@ -647,6 +647,43 @@ defmodule Oban do
     |> Query.retry_job(job_id)
   end
 
+  @doc """
+  Retries all discarded and retryable jobs. For more complex mass retries, see `retry_all_jobs/2`.
+
+  ## Example
+
+      Oban.retry_all_jobs()
+      :ok
+  """
+  @doc since: "2.8.0"
+  @spec retry_all_jobs :: :ok
+  def retry_all_jobs() do
+    __MODULE__
+    |> config()
+    |> Query.retry_all_jobs()
+  end
+
+  @doc """
+  Retries all jobs that match on the given queryable.
+
+  ## Example
+
+      # Retries all jobs with priority 0
+      Oban.Job |> Ecto.Query.where(priority: 0) |> Oban.retry_all_jobs()
+      :ok
```

Comment:
This example looks a bit dangerous to me. 

It makes me think that only failed jobs will be retried, but actually it also retries completed or even **executing** jobs with prio 0.

Reply:
That's an excellent point. Retrying `executing` jobs would cause a lot of issues. The `retry_all_jobs` function would need to apply a `status` condition, which may defeat the point of passing in a query entirely.